chips/README.md

```diff
@@ -0,0 +1,302 @@
+`leadingIcon: (@Composable () -> Unit)? = null` | Optional Content to be displayed before label into the Chip
+
+_In your project :_
+```kotlin
+VitaminChips.Action(
+    label = "Share",
+    onClick = {},
+    leadingIcon = {
+        Icon(
+            painter = painterResource(id = R.drawable.ic_vtmn_share_line),
+            contentDescription = null
+        )
+    }
+)
+```
+
+### Filter Chip
+
+Filter chips are used as filters for list or content.
+Multiple chips can be selected or unselected.
+A selected Filter Chip displays a tick as start content.
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun Filter(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        selected: Boolean = false,
+        colors: VitaminSelectableChipColors = VitaminChipDefaults.selectable(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit,
+        selectedIcon: (@Composable () -> Unit) = { VitaminChipButtons.Selected() }
+    ) 
+}
+
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`selected: Boolean,` | True if the Chip is selected, otherwise false
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`selected: Boolean = true` | If True, leading tick Icon will be displayed.
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripple: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the Chip is clicked
+`selectedIcon: (@Composable () -> Unit) = { VitaminChipButtons.Selected() }` | Icon Content to be displayed before label into the Chip when selected. Default is a Tick Icon
+
+_In your project :_
+```kotlin
+VitaminChips.Filter(
+    label = "Climbing",
+    selected = true,
+    sizes = VitaminChipSizes.small(),
+    onClick = { 
+        // callback when chip is clicked
+    }
+)
+```
+
+### Input Chip
+
+Input chips represent complex information used in fields or filters, like person, place, thing, attributes, or text.
+An icon or an image can be used to give information about the kind of information.
+
+#### Text alone 
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun Input(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminChipRipples.default(),
+        onClick: () -> Unit,
+        trailingIcon: (@Composable VitaminChipButtons.() -> Unit) = { VitaminChipButtons.Close(contentDescription = null) }
```

Comment:
If the default value is the `Close` icon, we should provide the content description.

Reply:
Same comment for all other `Close` component as default value

Reply:
I removed this input because we don't want it customizable for now